templates/prelogin/_sections/home/about_dimagi.html

```diff
@@ -37,7 +37,7 @@ <h2 class="text-center" style="margin-bottom: 1em;">
                     {% endblocktrans %}
                 </p>
                 <p>
-                    <a href="https://groups.google.com/forum/#!forum/commcare-users/join"
+                    <a href="https://forum.dimagi.com/"
                        class="btn btn-primary-dark btn-lg btn-block">
                         {% trans 'Visit CommCare Users' %}
```

Comment:
Should this say `Visit CommCare Users Forum`?

Reply:
yep, good call. fixed surrounding text as well